src/utils/utils.ts

```diff
@@ -60,6 +60,12 @@ import { SignMode } from "cosmjs-types/cosmos/tx/signing/v1beta1/signing";
 import { Int53 } from "@cosmjs/math";
 import { TxRaw } from "cosmjs-types/cosmos/tx/v1beta1/tx";
 import { fromBase64 } from "@cosmjs/encoding";
+import { MultiChainMsg, RouteResponse } from "@/solve";
+
+const NEUTRON_SWAP_GAS_AMOUNT = 1_200_000;
+const NEUTRON_TRANSFER_GAS_AMOUNT = 1_200_000;
```

Comment:
So I think the fee_swap is what costs much more gas, so technically an ibc_transfer can happen without a fee_swap if the input denom is the fee denom.

This seems fine to me, just under those conditions we are providing more gas than needed.